src/chart/graph/forceLayout.js

```diff
@@ -84,11 +85,16 @@ export default function (ecModel) {
                     d = (edgeLength[0] + edgeLength[1]) / 2;
                 }
                 var edgeModel = edge.getModel();
+                var curveness = zrUtil.retrieve3(
+                    edge.getModel().get('lineStyle.curveness'),
```

Comment:
`edge.getModel()` could be replaced by the variable `edgeModel` declared in previous code line.